client/src/public/ts/rendering.ts

```
@@ -127,8 +139,12 @@ function render(elapsedMilliseconds: number) {
     renderBeautifulScoreDisplay();
   } else {
     const score = variables.currentGameClientSide.shownScore;
-    stageItems.textSprites.scoreText.text =
-      parseInt(score).toLocaleString("en-US") || "0";
+    if (isNaN(score)) {
+      stageItems.textSprites.scoreText.text = "0";
+    } else {
+      stageItems.textSprites.scoreText.text =
+        parseInt(score.toString()).toLocaleString("en-US") || "0";
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)